common/lib/capa/capa/inputtypes.py

```diff
@@ -355,7 +358,7 @@ def get_html(self):
 
         context = self._get_render_context()
 
-        html = self.capa_system.render_template(self.template, context)
+        html = self.capa_system.render_template(self.template, context).strip()
```

Comment:
@muzaffaryousaf @ehteshamkafeel Here is the fix to `Argument 'element' has incorrect type` issue when we have `&amp` in xml.